lib/constants.py

```diff
@@ -71,4 +100,80 @@ def block_to_number(blockname: str) -> int:
   else:
     return block_name_to_int[blockname.upper()]
 
+def volkswagen_security_algo(level: int, seed: bytes, params=None) -> bytes:
```

Comment:
It's probably best if the _algo_ wrapper is moved back into the flasher code, as I'd like that to be self-documenting (retain all flashing concerns). Plus, that way the flasher can gain the Sa2SeedKey dependency back instead of the constants file. But, we can do that in a later commit, not a blocker for merge.

I do agree that the SA2 script should stay here, so when we move the method back, we'll want to pull the SA2 script back. That will let use add SA2 constants for S12 and S18.10 eventually too.